inst/json/table_data.json

```diff
@@ -3,10 +3,10 @@
     "intervention": "none",
     "netUse": 0,
     "irsUse": 0,
-    "casesAverted": 400,
-    "casesAvertedPer1000": 300,
-    "casesAvertedPer1000ErrorPlus": 16,
-    "casesAvertedPer1000ErrorMinus": 41,
+    "casesAverted": 0,
```

Comment:
this is all still fake data, just put some zeros in for no intervention to mimic real data